examples/controlnet/train_controlnet_flax.py

```diff
@@ -176,6 +167,7 @@ def save_model_card(repo_id: str, image_logs=None, base_model=str, repo_folder=N
 - text-to-image
 - diffusers
 - controlnet
+- jax-diffusers-event
```

Comment:
Cool, we just need to remember this is specific for the event and remove later.

Reply:
```suggestion
```
think we can remove this line here and keep it in the script on the event repo